src/main/java/net/minestom/server/FeatureFlagImpl.java

```diff
@@ -0,0 +1,33 @@
+package net.minestom.server;
+
+import net.minestom.server.registry.Registry;
+import net.minestom.server.utils.NamespaceID;
+import org.jetbrains.annotations.NotNull;
+
+public record FeatureFlagImpl(@NotNull Registry.FeatureFlagEntry registry) implements FeatureFlag {
```

Comment:
Add some documentation

Reply:
None of the other data-generated impl classes have documentation. This is also an internal class that I forgot to make package-private.